controllers/likedVideo.controller.js

```diff
@@ -0,0 +1,86 @@
+const User = require("../models/user-model");
+const LikedVideo = require("../models/likedVideo-model");
+
+const getLikedVideos = async (req, res) => {
+  const likedVideos = await LikedVideo.find({});
+  res.json({ success: true, likedVideos });
+};
+
```

Comment:
Its good to add a try catch block here too

Reply:
As Tanay mentioned that the common error handler could be used in most of the cases except where I need custom errors, so just wanted to implement it.